src/coreComponents/physicsSolvers/multiphysics/PhaseFieldFractureSolver.cpp

```diff
@@ -36,333 +34,88 @@ using namespace constitutive;
-        setDescription( "Total Mean Stress" );
-    } );
-  } );
-}
-
-void PhaseFieldFractureSolver::implicitStepSetup( real64 const & GEOSX_UNUSED_PARAM( time_n ),
-                                                  real64 const & GEOSX_UNUSED_PARAM( dt ),
-                                                  DomainPartition & domain )
-{
-  GEOSX_MARK_FUNCTION;
-  forDiscretizationOnMeshTargets( domain.getMeshBodies(), [&] ( string const &,
-                                                                MeshLevel & mesh,
-                                                                arrayView1d< string const > const & )
-  {
-
-    ElementRegionManager & elemManager = mesh.getElemManager();
-
-    ElementRegionManager::ElementViewAccessor< arrayView1d< real64 > > const totalMeanStress =
-      elemManager.constructViewAccessor< array1d< real64 >, arrayView1d< real64 > >( viewKeyStruct::totalMeanStressString() );
-
-    ElementRegionManager::ElementViewAccessor< arrayView1d< real64 > > oldTotalMeanStress =
-      elemManager.constructViewAccessor< array1d< real64 >, arrayView1d< real64 > >( viewKeyStruct::oldTotalMeanStressString() );
-
-    //***** loop over all elements and initialize the derivative arrays *****
-    forAllElemsInMesh( mesh, [ &]( localIndex const er, localIndex const esr, localIndex const k )
-    {
-      oldTotalMeanStress[er][esr][k] = totalMeanStress[er][esr][k];
-    } );
-  } );
+  m_couplingType = CouplingType::Sequential;
 }
 
-void PhaseFieldFractureSolver::implicitStepComplete( real64 const & GEOSX_UNUSED_PARAM( time_n ),
-                                                     real64 const & GEOSX_UNUSED_PARAM( dt ),
-                                                     DomainPartition & GEOSX_UNUSED_PARAM( domain ) )
-{}
-
-void PhaseFieldFractureSolver::postProcessInput()
-{
-  if( m_couplingTypeOption == CouplingTypeOption::FixedStress )
-  {
-    // For this coupled solver the minimum number of Newton Iter should be 0 for both flow and solid solver otherwise it
-    // will never converge.
-    SolidMechanicsLagrangianFEM &
-    solidSolver = this->getParent().getGroup< SolidMechanicsLagrangianFEM >( m_solidSolverName );
-    integer & minNewtonIterSolid = solidSolver.getNonlinearSolverParameters().m_minIterNewton;
-
-    PhaseFieldDamageFEM &
-    damageSolver = this->getParent().getGroup< PhaseFieldDamageFEM >( m_damageSolverName );
-    integer & minNewtonIterFluid = damageSolver.getNonlinearSolverParameters().m_minIterNewton;
-
-    minNewtonIterSolid = 0;
-    minNewtonIterFluid = 0;
-  }
-}
-
-void PhaseFieldFractureSolver::initializePostInitialConditionsPreSubGroups()
-{}
-
 PhaseFieldFractureSolver::~PhaseFieldFractureSolver()
 {
   // TODO Auto-generated destructor stub
 }
 
-void PhaseFieldFractureSolver::resetStateToBeginningOfStep( DomainPartition & domain )
+void PhaseFieldFractureSolver::mapSolutionBetweenSolvers( DomainPartition & domain, integer const idx )
```

Comment:
```suggestion
void PhaseFieldFractureSolver::mapSolutionBetweenSolvers( DomainPartition & domain, integer const solverType )
```
to make it clear that the index that is passed is the solver type, even if we don't have the `enum` defined in the base `CoupledSolver` class